tests/dbt/test_transformation.py

```diff
@@ -394,18 +393,28 @@ def test_statement(sushi_test_project: Project, runtime_renderer: t.Callable):
     assert context.target
     engine_adapter = context.target.to_sqlmesh().create_engine_adapter()
     renderer = runtime_renderer(context, engine_adapter=engine_adapter)
-    assert renderer(
-        "{% set test_var = 'SELECT 1' %}{% call statement('something', fetch_result=True) %} {{ test_var }} {% endcall %}{{ load_result('something').table }}",
-    ) == str(agate.Table([[1]], column_names=["1"], column_types=[agate.Number()]))
+    assert (
+        renderer(
+            "{% set test_var = 'SELECT 1' %}{% call statement('something', fetch_result=True) %} {{ test_var }} {% endcall %}{{ load_result('something').table }}",
+        )
+        == """| column | data_type |
+| ------ | --------- |
+| 1      | Integer   |
+"""
+    )
 
 
 def test_run_query(sushi_test_project: Project, runtime_renderer: t.Callable):
     context = sushi_test_project.context
     assert context.target
     engine_adapter = context.target.to_sqlmesh().create_engine_adapter()
     renderer = runtime_renderer(context, engine_adapter=engine_adapter)
-    assert renderer("{{ run_query('SELECT 1 UNION ALL SELECT 2') }}") == str(
-        agate.Table([[1], [2]], column_names=["1"], column_types=[agate.Number()])
+    assert (
+        renderer("{{ run_query('SELECT 1 UNION ALL SELECT 2') }}")
+        == """| column | data_type |
+| ------ | --------- |
+| 1      | Integer   |
+"""
```

Comment:
@crericha Not sure if this is something local to my env, but when I run the original expected result: 
```
str(
        agate.Table([[1], [2]], column_names=["1"], column_types=[agate.Number()])
```
I get, as shown above, just the number 1 in the output. Is this expected?